client.js

```diff
@@ -116,10 +116,15 @@ module.exports = class TelegramClient {
         }
         if (errorMessage.includes('FLOOD_WAIT')) {
           const seconds = errorMessage.split('_WAIT_')[1];
-          throw new Error(`Our servers busy talking to Telegram. Please try again in ${seconds} seconds`);
+          await delay((seconds * 1000) + 1);
+          return await this.request(...args);
         }
         throw new Error(errorMessage);
       });
+
+    // this is a simple fix to stop telegram from throwing flood errors, this wont work if requests are done in parralel
```

Comment:
i can think of a possibly better solve for this. should be an easy pattern to whip up.